.github/workflows/publish.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow publishes the dart and the flutter package to pub.dev, when a new release
+# or pre-release is published on GitHub.
+#
+# Just before publishing, the code is formatted.
+
+name: publish-release
```

Comment:
Could you
- remove all comments and empty lines
- rename this file to `release-manual.yml`
- change the workflow name to `name: release-manual`